### ISSUE TITLE
[XMPP] unable to remove jabber contacts

### ISSUE THREAD

**Issue by pravi** [loqui/im#184]
Loqui 0.2.4
jabber server autistici.org

it asks for confirmation, but fails to delete.

**Comment by Gioyik** [comment]
Could you provide us some logs?

**Comment by pravi** [comment]
@Gioyik is these some documentation on how to collect logs?

**Comment by aesedepece** [comment]
You have to enable "debug mode" from settings menu or run the following command from App Manager:
-  `App.settings.debug = true; App.smartupdate('settings')` for v0.2.4
-  `App.devsettings.debug = true; App.smartupdate('devsettings')` for v0.2.5 (master)

Then you can see logs in App Manager or by using the oldie `adb logcat | grep GeckoConsole`.

**Comment by pravi** [comment]
my usb ports are broken right now, so can't test on my ZTE Open. But I was trying it on fxos simulator and now I can't even open the profile page to delete the contact.

I go to contacts -> click on the contact I want to delete -> try to click on the top right icon

but nothing happens. I will try to collect the logs of this and post it.

**Comment by pravi** [comment]
ok, now I got to reproduce the error. The log output is,

13:29:48.506 Expected ':' but found ']'.  Declaration dropped. index.html
13:30:27.919 TypeError: this.connection is undefined xmpp.js:202

**Comment by aesedepece** [comment]
Is the contact of that chat in your roster?

**Comment by pravi** [comment]
yes, I added it in my phone's loqui (actually I mis spelt the address so I wanted to remove). I could see it on emulator's loqui as well. So it is indeed present in the roster, I also checked it in empathy on the desktop.

**Comment by TitanNano** [comment]
Is this still relevant?

**Comment by pravi** [comment]
Yes, I still can't remove contacts. V0.4.0a
